### DISCUSSION TITLE
Featured Projects section on the homepage

### DISCUSSION THREAD

**Discussion by woodiertexas** [modrinth/.github#21] [Feature requests]
### Is your feature request related to a problem? Please describe.

There is no "Featured Projects" on the modrinth homepage.

### Describe the solution you'd like

A "Featured Project" section containing a randomized assortment of modrinth projects in a grid. Could contain up to 64 random projects.

### Additional context

something similar to this except actually better:
![](https://cdn.discordapp.com/attachments/783091855616901200/954878396360454184/unknown.png)

**Comment by woodiertexas** [comment]
some alternatives:

#426 - random order sorting in mods page
a "take me to a random selection of mods" which takes you to the mods page sorted in random order
a dedicated page for the featured projects assortment

**Comment by MulverineX** [comment]
@wafflecoffee what would be a bad thing about having a randomized projects listing? (re: your downvote)

**Comment by PssbleTrngle** [comment]
Not sure if this should be one the home page, but having a specific page to browse "random" mods would be nice sometimes.

It's not something users would want to do often, that's why I think it should not be on the landing page, but a link/button somewhere in the mods search taking you there could be nice.

**Comment by triphora** [comment]
@MulverineX Randomized is fine, and actually already somewhat present on [the rewrite](https://rewrite.modrinth.com). I think the issue would come if we were to curate it, which I think is what CF does.

**Comment by triphora** [comment]
This is implemented as a carousel of random projects on the homepage
